examples/spl_composites/com.ibm.streamsx.testing.python/testCustom.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+
+from streamsx.topology.topology import *
+from streamsx.topology.tester import Tester
+import streamsx.spl.op as op
+import streamsx.spl.toolkit as tk
+import numpy as np
+
+# TestCustomStandalone class for driving the test in standalone mode
+class TestCustomStandalone(unittest.TestCase):
+    def setUp(self):
+        ''' Set up to run test as Standalone'''
+        Tester.setup_standalone(self)
+    def _add_toolkits(self, topo):
+        ''' Add required toolkits for test to run '''
+        tk.add_toolkit(topo, '../com.ibm.streamsx.testing.app')
+        tk.add_toolkit(topo, '../com.ibm.streamsx.testing.spl')
+    def test_op(self):
+        ''' Create topology to drive the test from com.ibm.streamsx.testing.spl '''
+        topo = Topology()
+        self._add_toolkits(topo)
+
+        '''Set up parameter to call the test composite'''
+        params = {'factor':3}
+
+        ''' Call the test composite'''
+        testStream = op.Source(topo, 'com.ibm.streamsx.testing.spl::TestMultiplyBy', 'tuple<int32 result>', params=params)
+
+        ''' Convert the SPLStream to Python Stream so we can work with the data in the tester '''
```

Comment:
The tester can handle SPL/structured stream so it's not clear why you are converting here.

Reply:
Can you show how?  I couldn't get it to work  without going down this path.